### DISCUSSION TITLE
Want to stream with delay

### DISCUSSION THREAD

**Discussion by CyberSensei1** [easytarget/esp32-cam-webserver#262] [Q&A] [answered]
Hi, i want to stream my camera with delay. I want client will watch stream 3-5 seconds behind i tried a little but i am not familiar with esp32cam library what is the best way to do that?

**Comment by easytarget** [comment, marked as answer]
No, there is no way to do this on the camera side, it would need to store several seconds worth of video, not enough memory etc..
